.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: erlang
+dist: trusty
```

Comment:
Presets from "matrix" by default run on 12.04 which is unable to compile one of the new dependencies due to lack of C++11 support in GCC.